cypress/integration/visual-test/list-test.spec.js

```diff
@@ -0,0 +1,47 @@
+import { baseURL, checkEyes } from '../../util';
+describe('Visual Regression Testing', () => {
+  before(() => {
+    cy.eyesOpen({
+      appName: 'Clarity',
+      testName: 'list',
+      browser: { width: 1024, height: 768, name: 'chrome' },
+    });
+  });
+  after(() => {
+    cy.eyesClose();
+  });
+  it('check lists-ul', () => {
```

Comment:
This set of tests (and others which are just navigate and checkEyes) could be shortened to a list of urls that we loop over something like this

```
const tests = ['lists-ul', 'lists-unstyled', 'lists-ol', lists-mixed', 'lists-compact', 'lists-in-cards', 'old-lists-in-cards'];
tests.forEach((test) => {
  it(`check ${test}`, () => {
    cy.visit(`lists/${test}`);
    checkEyes(test);
  });
});
```

Reply:
done